### ISSUE TITLE
about feature extraction of THUMOS14

### ISSUE THREAD

**Issue by menghuaa** [Pilhyeon/Learning-Action-Completeness-from-Points#7]
Excuse me, what methuod do you use to extract the feature of thumos14? Is it I3D?

**Comment by Pilhyeon** [comment]
Yes, we used the two-stream I3D networks pre-trained on Kinetics-400, as stated in the paper.

**Comment by menghuaa** [comment]
> Yes, we used the two-stream I3D networks pre-trained on Kinetics-400, as stated in the paper.

ok，thank you.You said in your paper that you take 16 frames of video as a segment, and then extract features. But when I output the shape (nxdimension) of the feature, n is not equal to the total number of frames /16. Why is this?

**Comment by Pilhyeon** [comment]
It is probably because we set the fps of all videos in THUMOS'14 to 25 before extracting the features.

**Comment by menghuaa** [comment]
> 

Yes，that makes sense！Thank you very much！

**Comment by PHDJieFu** [comment]
> > 
> 
> Yes，that makes sense！Thank you very much！

Hello，I am a PhD student from China. I am also interested in this work now. I have some issues about this work. Can we make a discussion? My WeChat ID is fj_snbc.
